### PR TITLE
refac: rewriting charge tests

### PR DESCRIPTION
**Pull request by energinet-ajw**
The follwing two tests are rewritten (part 2).
* Reading charges contain relevant columns
* Reading charges joins on charge_key 

The precending tests follow a different format than now.

https://app.zenhub.com/workspaces/mandalorian-5fd22a1a59e4740018f5ab5a/issues/gh/energinet-datahub/green-energy-hub/470

### CODE REVIEW

File: source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py [hunk 1]
```diff
@@ -15,7 +15,12 @@
 from decimal import Decimal
 from datetime import datetime
 from unittest.mock import patch
+
+import pytest
+from pyspark import Row
 from pyspark.sql import SparkSession
```
Comment (BjarkeMeier) [inline source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py]: Oh sweet. You're following [pep8](https://peps.python.org/pep-0008/#imports) regarding imports 👍

Comment (energinet-ajw) [reply source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py]: I am? Nice :)

File: source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py [hunk 2]
```diff
@@ -43,7 +48,7 @@ def _create_charge_master_data_row(
     resolution: str = DEFAULT_RESOLUTION,
     from_date: datetime = DEFAULT_FROM_DATE,
     to_date: datetime = DEFAULT_TO_DATE,
-) -> dict:
+) -> Row:
     row = {
```
Comment (BjarkeMeier) [inline source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py]:
```suggestion
    row = Row(
```

File: source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py [hunk 3]
```diff
@@ -54,7 +59,7 @@ def _create_charge_master_data_row(
         Colname.from_date: from_date,
         Colname.to_date: to_date,
     }
-    return row
+    return Row(**row)
```
Comment (BjarkeMeier) [inline source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py]: In conjuction with previous suggestion
```suggestion
    return row
```

Comment (BjarkeMeier) [reply source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py]: Applies to other occurrences too.

File: source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py [hunk 4]
```diff
@@ -129,3 +134,32 @@ def test__read_changes__returns_expected_joined_row_values(
     assert actual_row[Colname.to_date] == DEFAULT_TO_DATE
     assert actual_row[Colname.observation_time] == DEFAULT_OBSERVATION_TIME
     assert actual_row[Colname.metering_point_id] == DEFAULT_METERING_POINT_ID
+
+
+@patch.object(calculation_input, DeltaTableReader.__name__)
```
Comment (BjarkeMeier) [inline source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py]: Nice - supporting tool-assisted renamings 👍

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**